src/main/java/com/google/devtools/common/options/OptionsParserImpl.java

```diff
@@ -621,8 +621,8 @@ private boolean shouldIgnoreOption(OptionDefinition optionDefinition) {
       try {
         optionDefinition.getField().set(optionsInstance, value);
       } catch (IllegalArgumentException e) {
-        throw new IllegalStateException(
-            String.format("Unable to set %s to value '%s'.", optionDefinition, value), e);
+        // May happen when a boolean option got a string value. Just ignore this error without
```

Comment:
Just curious, how does Bazel report wrong value of the flag?

Reply:
It says:
```
bazel build --verbose_failures=/foo
(...)
ERROR: While parsing option --verbose_failures=/foo: '/foo' is not a boolean
```

Reply:
Perfect, thanks!